probe-rs/targets/esp32c3.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 name: esp32c3
 manufacturer:
-  cc: 0x12
-  id: 0x12
+  cc: 0x0B
```

Comment:
👀

Reply:
This was indeed a bug, see https://github.com/probe-rs/probe-rs/pull/1779